academic-services/knowledge-exploration-service/reference-get-interpret.md

```diff
@@ -456,3 +456,84 @@ Date: Thu, 30 Jan 2020 01:08:52 GMT
     "timed_out": false
 }
 ```
+
+### Map academic reference to paper
+
+This example maps a raw academic reference string to the academic paper that most closely maps to the metadata present in the reference string.
+
+The reference string we will be using:
+
+```
+Sinha, Arnab, et al. "An Overview of Microsoft Academic Service (MAS) and Applications." Proceedings of the 24th International Conference on World Wide Web, 2015, pp. 243–246.
+```
+
+#### Request headers
+
+```http
+GET /interpret?query=Sinha,%20Arnab,%20et%20al.%20"An%20Overview%20of%20Microsoft%20Academic%20Service%20(MAS)%20and%20Applications."%20Proceedings%20of%20the%2024th%20International%20Conference%20on%20World%20Wide%20Web,%202015,%20pp.%20243–246.&count=1&entityCount=1&attributes=Id,DN,Y,AA.DAuN,VFN HTTP/1.1
+Host: makesexample.westus.cloudapp.azure.com
```

Comment:
>makesexample.westus.cloudapp.azure.com [](start = 6, length = 38)

guessing this is fake?

Reply:
Yes, it's the same one I use for all MAKES examples

---
In reply to: [398024088](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/135#discussion_r398024088) [](ancestors = 398024088)